### PR TITLE
doc(release): prepare release notes for OSS 23.10

### PR DESCRIPTION
**Pull request by pkippes**
"web 23.10.10
gorgone 23.10.4"

## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [x] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx [hunk 1]
```diff
@@ -22,6 +22,36 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
...
+
+</details>
+
+<details>
+  <summary>Bug fixes</summary>
+
+- [Authentication] Fixed issue in SSO authentication.
+- [API] Fixed a visibility issue on UserFilter endpoint.
+- [API] Fixes an issue where 500 error was returned when a host didn't have alias.
+- [CLAPI] Fixed a decoding issue on HTML entities export.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx:42]:
```suggestion
- [CLAPI] Fixed a decoding issue affecting the export of HTML entities.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx [hunk 2]
```diff
@@ -22,6 +22,36 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
...
+- [CLAPI] While exporting services and hosts, duplicates were generated.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx:43]:
```suggestion
- [CLAPI] Duplicates are no longer generated when exporting services and hosts.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx [hunk 3]
```diff
@@ -22,6 +22,36 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
...
+
+- [Resource Status] Improved update of filters by removing useless dialog
+
+</details>
+
+<details>
+  <summary>Bug fixes</summary>
+
+- [Authentication] Fixed issue in SSO authentication.
+- [API] Fixed a visibility issue on UserFilter endpoint.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx:40]:
```suggestion
- [API] Fixed a visibility issue with the UserFilter endpoint.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx [hunk 4]
```diff
@@ -22,6 +22,36 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
...
 ## Centreon Web
 
+### 23.10.10
+
+Release date: `March 12, 2024`
+
+<details open>
+  <summary>Enhancements</summary>
+
+- [Resource Status] Improved update of filters by removing useless dialog
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx:32]:
```suggestion
- [Resource Status] Improved the update of filters by removing unnecessary dialog box.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx [hunk 5]
```diff
@@ -22,6 +22,36 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
...
+
+<details>
+  <summary>Bug fixes</summary>
+
+- [Authentication] Fixed issue in SSO authentication.
+- [API] Fixed a visibility issue on UserFilter endpoint.
+- [API] Fixes an issue where 500 error was returned when a host didn't have alias.
+- [CLAPI] Fixed a decoding issue on HTML entities export.
+- [CLAPI] While exporting services and hosts, duplicates were generated.
+- [Dashboards - Map widget] Fixed Map, Geoview and perspective save.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx:44]: To be improved

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by rsadik-ext — APPROVED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx [hunk 6]
```diff
@@ -22,6 +22,36 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
...
+
+<details>
+  <summary>Bug fixes</summary>
+
+- [Authentication] Fixed issue with SSO authentication.
+- [API] Fixed a visibility issue with the UserFilter endpoint.
+- [API] Fixed an issue where an error 500 was returned when a host had no alias.
+- [CLAPI] Fixed a decoding issue affecting the export of HTML entities.
+- [CLAPI] Duplicates are no longer generated when exporting services and hosts.
+- [Dashboards - Map widget] Fixed Map, Geoview and perspective save.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx:44]:
```suggestion
- [Dashboards - Map widget] Fixed an issue where Map, Geoview and perspectives couldn't be saved.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx [hunk 7]
```diff
@@ -22,6 +22,36 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
...
+
+<details>
+  <summary>Bug fixes</summary>
+
+- [Authentication] Fixed issue with SSO authentication.
+- [API] Fixed a visibility issue with the UserFilter endpoint.
+- [API] Fixed an issue where an error 500 was returned when a host had no alias.
+- [CLAPI] Fixed a decoding issue affecting the export of HTML entities.
+- [CLAPI] Duplicates are no longer generated when exporting services and hosts.
+- [Dashboards - Map widget] Fixed an issue where Map, Geoview and perspectives couldn't be saved.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx:44]:
```suggestion
- [Dashboards - Map widget] Fixed an issue where Maps, Geoviews and perspectives couldn't be saved.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx [hunk 8]
```diff
@@ -22,6 +22,36 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
...
 ## Centreon Web
 
+### 23.10.10
+
+Release date: `March 13, 2024`
+
+<details open>
+  <summary>Enhancements</summary>
+
+- [Resource Status] Improved the update of filters by removing unnecessary dialog box.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx:32]:
```suggestion
- [Resource Status] Improved the update of filters by removing an unnecessary dialog box.
```

File: versioned_docs/version-23.10/releases/centreon-os.mdx [hunk 9]
```diff
@@ -23,6 +23,36 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Dashboards - Map widget] Fixed Map, Geoview and perspective save.
+
+</details>
+
+<details open>
+  <summary>Security fixes</summary>
+
+-	[Security] Improved Centreon Gorgone allowed commands
+
+</details>
```
Comment (cg-tw) [inline versioned_docs/version-23.10/releases/centreon-os.mdx:54]:
```suggestion
<details open>
  <summary>Enhancements</summary>

- [Resource Status] Improved the update of filters by removing an unnecessary dialog box.

</details>

<details>
  <summary>Bug fixes</summary>

- [Authentication] Fixed issue with SSO authentication.
- [API] Fixed a visibility issue with the UserFilter endpoint.
- [API] Fixed an issue where an error 500 was returned when a host had no alias.
- [CLAPI] Fixed a decoding issue affecting the export of HTML entities.
- [CLAPI] Duplicates are no longer generated when exporting services and hosts.
- [Dashboards - Map widget] Fixed an issue where Maps, Geoviews and perspectives couldn't be saved.

</details>

<details open>
  <summary>Security fixes</summary>

- [Security] Improved allowed commands for Centreon Gorgone.

</details>
```

File: versioned_docs/version-23.10/releases/centreon-os.mdx [hunk 10]
```diff
@@ -411,6 +441,19 @@ Release date: `October 30, 2023`
...
+### 23.10.4
+
+Release date: `March 13, 2024`
+
+<details>
+  <summary>Bug fixes</summary>
+
+- Added the Auto-Discovery patterns to the default commands whitelist.
+- Improved list of allowed commands for Auto-Discovery.
+- [Packaging] Fixed missing perl dependency for centreon-web installation on debian 11.
```
Comment (cg-tw) [inline versioned_docs/version-23.10/releases/centreon-os.mdx:453]:
```suggestion
- Added the Auto Discovery patterns to the default commands whitelist.
- Improved list of allowed commands for Auto Discovery.
- [Packaging] Fixed missing perl dependency for centreon-web installation on Debian 11.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by rsadik-ext — APPROVED** [review]

**Review by rsadik-ext — APPROVED** [review]

**Review by rsadik-ext — APPROVED** [review]

**Pull request merged**